src/algebra/ne_zero.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2021 Eric Rodriguez. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Rodriguez
 -/
-import logic.basic
+import algebra.group.defs
```

Comment:
Sorry, this is not allowed. You will need to put the lemmas elsewhere.

Reply:
Where should I put these `@[field_simps]` lemmas?